content/en/bits_ai/_index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Bits AI
+kind: documentation
+disable_toc: false
+private: true
+is_beta: true
+further_reading:
+    - link: "https://www.datadoghq.com/product/platform/bits-ai/"
+      tag: "Product page"
+      text: "Bits AI"
+    - link: "https://www.datadoghq.com/blog/datadog-bits-generative-ai/"
+      tag: "Blog post"
+      text: "Introducing Bits AI, Your New DevOps Copilot"
+    - link: "https://www.datadoghq.com/about/latest-news/press-releases/datadog-announces-bits-an-ai-assistant-to-help-engineers-quickly-resolve-application-issues/"
+      tag: "Press release"
+      text: "Bits AI announcement"
+---
+
+<div class="alert alert-info">Bits AI is in private beta.</div>
+
+Bits AI is a generative AI interface that helps you identify, remediate, and summarize issues in your applications and infrastructure. It surfaces faulty deployments, Watchdog anomalies, incidents, alerts, and more.
+
+## Features
+
+### Query data in natural language
+
+Bits AI supports natural language querying for logs, APM traces, infrastructure data, and cloud cost.
+
+In Datadog or in Slack, you can ask Bits AI questions such as
```

Comment:
You can also ask Bits AI to tell you about the health and ownership of your services, and retrieve Datadog resources related to those services.